tests/Feature/Community/Actions/BuildTrendingGamesActionTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Community\Actions;
+
+use App\Community\Actions\BuildTrendingGamesAction;
+use App\Community\Data\TrendingGameData;
+use App\Enums\Permissions;
+use App\Models\Game;
+use App\Models\System;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class BuildTrendingGamesActionTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testItReturnsEmptyDataWhenNoActivePlayers(): void
+    {
+        // Act
+        $result = (new BuildTrendingGamesAction())->execute();
+
+        // Assert
+        $this->assertCount(0, $result);
+    }
+
+    public function testItReturnsTrendingGames(): void
+    {
+        // Arrange
+        $system = System::factory()->create();
+        $game1 = Game::factory()->create(['ConsoleID' => $system->id, 'Title' => 'most_popular']);
+        $game2 = Game::factory()->create(['ConsoleID' => $system->id, 'Title' => 'second_most_popular']);
+        $game3 = Game::factory()->create(['ConsoleID' => $system->id, 'Title' => 'third_most_popular']);
+        $game4 = Game::factory()->create(['ConsoleID' => $system->id, 'Title' => 'fourth_most_popular']);
+        $game5 = Game::factory()->create(['ConsoleID' => $system->id, 'Title' => 'fifth_most_popular']);
```

Comment:
Recommend inserting these out-of-order to also verify the sorting.

Reply:
Updated in latest.